### ISSUE TITLE
Cura download URL changend

### ISSUE THREAD

**Issue by BruegelN** [caskroom/homebrew-cask#11424]
The url used in cura.rb formula changed to https://ultimaker.com/en/software/download/23/Cura-15.02.1-MacOS.dmg so line 5 should be changed to:
```ruby
  url "https://ultimaker.com/en/software/download/23/Cura-#{version}-MacOS.dmg"
```
the old (/ invalid) one is : 
```ruby
  url "http://software.ultimaker.com/current/Cura-#{version}-MacOS.dmg"
```

which gets me this error:
```bash
==> Downloading http://software.ultimaker.com/current/Cura-15.02.1-MacOS.dmg

curl: (22) The requested URL returned error: 404 Not Found
Error: Download failed on Cask 'cura' with message: Download failed: http://software.ultimaker.com/current/Cura-15.02.1-MacOS.dmg
```
Because I don't if I should create a new pull request because of this single line I created at least a gist under : 
https://gist.github.com/BruegelN/d6a692fe3555d33fcc69

I replaced my ```/usr/local/Library/Taps/caskroom/homebrew-cask/Casks/cura.rb``` with the one from the gist and it worked just fine with ```brew cask install cura``` :smiley: .

Cheers
Nico

**Comment by vitorgalvao** [comment]
Yes, you should create a PR if you’re able. Why not?

**Comment by BruegelN** [comment]
Created PR #11437 with changed URL :smiley: 
So I can close this issue, there is now reason to wait untill pull request is accepted, is it ?

Cheers
Nico

**Comment by vitorgalvao** [comment]
Yes. You could also, on that PR, include “Closes #11424” or “Fixes #11424”, and it would close this issue automatically, when merged.
